TicTacToeConsoleApp/Program.cs

```diff
@@ -0,0 +1,180 @@
+﻿namespace TicTacToeConsoleApp
+{
+    public class Program
+    {
+        private static readonly Dictionary<string, string> _board = new();
+
+        private static Dictionary<int, string> _players = new Dictionary<int, string>
+        {
+                { 1, "Х" },
+                { 2, "О" }
+        };
+
+        private static string _currentPlayer;
+
+        private static readonly int _cellsNumber = 3;
+
+        private static void Main()
+        {
+            _currentPlayer = _players[1];
```

Comment:
you don't need this variable.
Use int _currentPlayerIndex and and get the desired player’s representations from the array.
var player1 = _players[0]
var player2 = _players[1]
var player3 = _players[2]
var player4 = _players[3]